client/src/components/Index/IndexPage.js

```diff
@@ -105,7 +105,7 @@ export default function IndexPage() {
                   return (
```

Comment:
The logic can be improved a bit here using ternary operator, try optimising it using only one return statement.

Reply:
Okay, it will be improved in the next PR.
Thanks for pointing out.